### ISSUE TITLE
[text] not working after update

### ISSUE THREAD

**Issue by chakibchemso** [spicetify/spicetify-themes#1103]
**Prerequisites**
Before opening an issue, make sure
1. v2.37.1
2. 1.2.44.405.g81fd6352
3. using store version
4. YES

**Describe the bug**
covers are messed up in main page.

**To Reproduce**
1. Open spotify

**Expected behavior**
1. working theme

**Screenshots**
- All
![image](https://github.com/user-attachments/assets/21e1036b-56f8-4f87-b747-6086897c6e73)
- Music
![image](https://github.com/user-attachments/assets/19a3ab6e-46c9-4640-900c-df5453762bd9)
- This is not as it used to be, cant remember exactly but, im sure the bg was blended perfectly.
![image](https://github.com/user-attachments/assets/7a467535-b8ce-44ad-85db-b30606c8f8b8)

**Logs**
```console
 document.domain mutation is ignored because the surrounding agent cluster is origin-keyed.
(anonymous) @ index.html:1
CircularSp-Book.woff2:1 
        
        
       Failed to load resource: net::ERR_NAME_NOT_RESOLVED
CircularSp-Bold.woff2:1 
        
        
       Failed to load resource: net::ERR_NAME_NOT_RESOLVED
CircularSpTitle-Bold.woff2:1 
        
        
       Failed to load resource: net::ERR_NAME_NOT_RESOLVED
CircularSpTitle-Black.woff2:1 
        
        
       Failed to load resource: net::ERR_NAME_NOT_RESOLVED
4151.css:1 
        
        
       Failed to load resource: net::ERR_NAME_NOT_RESOLVED
favicon.ico:1 
        
        
       Failed to load resource: net::ERR_NAME_NOT_RESOLVED
3vendor~xpui.js:1 ButtonTertiary must have an aria-label or aria-labelledby attribute defined when using the iconOnly prop in order to be accessible.
l @ vendor~xpui.js:1
extension.js:8 Initializing Spicetify Marketplace v1.0.2
extension.js:8 Loaded Marketplace extension
gew1-spclient.spotify.com/remote-config-resolver/v3/configuration:1 
        
        
       Failed to load resource: the server responded with a status of 404 ()
vendor~xpui.js:1 provider:transport resolve error k Object
h @ vendor~xpui.js:1
index.html:1 Uncaught (in promise) k
xpui.js:1 cosmos-connector:  fetchAdForSlot endpoint=%o sp://ads/v1/ads/hpto
xpui.js:1 Uncaught (in promise) InternalServerError: GET request to sp://ads/v1/ads/hpto failed with status code 500
    at c.onSuccess (xpui.js:1:2101530)
    at onSuccess (xpui.js:1:2101344)
16The resource <URL> was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSpTitle-Bold.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSpTitle-Black.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSp-Bold.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSp-Book.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSpTitle-Bold.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSpTitle-Black.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSp-Bold.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSp-Book.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSpTitle-Bold.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSpTitle-Black.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSp-Bold.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSp-Book.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSpTitle-Bold.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSpTitle-Black.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSp-Bold.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
index.html:1 The resource https://xpui.app.spotify.com/fonts/CircularSp-Book.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
```

**Specifics (please complete the following information):**
 - OS: Win11
 - Spotify version 1.2.44.405.g81fd6352
 - Spicetify version v2.37.1
 - text

**Comment by RWCoding58** [comment]
Same issue here

**Comment by outoffit** [comment]
also haven't been able to change color schemes for dribbblish or text from marketplace (even though there's still pictures of them in the description). powershell says configs changed & color scheme added and config-xpui.ini says my color scheme is the one I added but my theme (text) is still the black and green default spotify one.

**Comment by AtvouzX** [comment]
> also haven't been able to change color schemes for dribbblish or text from marketplace (even though there's still pictures of them in the description). powershell says configs changed & color scheme added and config-xpui.ini says my color scheme is the one I added but my theme (text) has the same black and green default spotify colors.

If you installed marketplace, you could change it from there, next to the search bar
![image](https://github.com/user-attachments/assets/9788f7f9-5a7a-45fc-8db2-b6086c78a6f8)

**Comment by outoffit** [comment]
@AtvouzX I didn't know that exists, thanks. 

I already switched back to outside marketplace to try the traditional way but the only reason I switched to marketplace was bc I couldn't change color schemes from the "spicetify-themes" folder before. This time I did everything the same but the themes folder appears to be in the correct location this time when before it was in "User > Name" directory instead of "spicetify" directory.

but I need to try again if I can change it that way via marketplace.

**Comment by octo-kumo** [comment]
![image](https://github.com/user-attachments/assets/395ed516-7a56-4c8b-b489-364e76a71d40)
![image](https://github.com/user-attachments/assets/226601d8-142a-4402-81a1-a68146136c27)
The controls are also not like before.

**Comment by chakibchemso** [comment]
- happened again on latest 1.2.53 @rxri @darkthemer 
- ![image](https://github.com/user-attachments/assets/fe362712-8a76-4515-9bea-e00fa5a38e71)

**Comment by rxri** [comment]
Do not ping me. I'm not responsible for text theme. This issue was closed automatically due to merging #1122 months ago.
